setup project for travis build

zaro0508 opened the pull request.

zaro0508 committed:
setup project for travis build

syoung-smallwisdom commented on CRFModuleValidation.xcodeproj/project.pbxproj:
```diff
@@ -653,7 +653,7 @@
 				CLANG_ENABLE_MODULES = YES;
 				CODE_SIGN_ENTITLEMENTS = CRFModuleValidation.entitlements;
-				"CODE_SIGN_IDENTITY[sdk=iphoneos*]" = "iPhone Developer";
+				"CODE_SIGN_IDENTITY[sdk=iphoneos*]" = "iPhone Distribution";
```
In order for debugging to work, this needs to match the developer profile and *not* the distribution profile.

syoung-smallwisdom commented on CRFModuleValidation.xcodeproj/project.pbxproj:
```diff
@@ -666,8 +666,8 @@
-				PROVISIONING_PROFILE = "acddad29-8401-4a1d-81db-36c1d3b4fa63";
-				PROVISIONING_PROFILE_SPECIFIER = "match Development org.sagebase.crfModuleApp";
+				PROVISIONING_PROFILE = "859e989d-f740-4844-b424-721f77405886";
+				PROVISIONING_PROFILE_SPECIFIER = "XC iOS: org.sagebase.crfModuleApp";
```
Needs to be the developer profile when building for debug.

syoung-smallwisdom reviewed (changes requested):
The developer profile needs to remain for debug. Only for the released production app, should the distribution profile be used. Otherwise, looks good.

zaro0508 committed:
Needs to be the developer profile when building for debug

zaro0508 committed:
Move tests to post merge build

Pre merge testings has been disabled because Travis does not support
private data in PR builds.  Testing can only be done post merge.

syoung-smallwisdom reviewed (approved):
Thanks!
